references/cli-man-page/unsupported-os.md

```diff
@@ -0,0 +1,49 @@
+# Unsupported Operating Systems
```

Comment:
```suggestion
# Alternative Installation Steps
```
it's not exactly "unsupported" if we support it, it's more like a different way to install...WDYT?

Reply:
hmm, fair... let me rename this then